### PR TITLE
Export traces less frequently

### PR DESCRIPTION
**Pull request by RebeccaMahany**
The exporter already has a maximum batch size and maximum queue size configured, but we were never hitting the maximum batch size because the batch timeout is set very low by default, enforcing shipping batches every 5 seconds. This probably isn't necessary for our usecase, so I've increased the batch timeout to a minute.

### CODE REVIEW

**Review by directionless — DISMISSED** [review]
Do we want control server adjustability here?

Or the pattern around uptime?

**Review by James-Pickett — DISMISSED** [review]

**Comment by RebeccaMahany** [comment]
> Do we want control server adjustability here?
> 
> Or the pattern around uptime?

I've updated so it's a control server flag!

File: pkg/agent/flags/keys/keys.go [hunk 1]
```diff
@@ -45,6 +45,7 @@ const (
 	UpdateDirectory                FlagKey = "update_directory"
 	ExportTraces                   FlagKey = "export_traces"
 	TraceSamplingRate              FlagKey = "trace_sampling_rate"
+	TraceBatchTimeout              FlagKey = "trace_batch_timeout"
```
Comment (directionless) [inline pkg/agent/flags/keys/keys.go:48]: Do we need this as an exported flag?

**Review by directionless — APPROVED** [review]
If you think this should be exposed as a flag, okay.

File: pkg/agent/flags/keys/keys.go (hunk 1 above)
Comment (RebeccaMahany) [reply pkg/agent/flags/keys/keys.go:48]: You're right -- removed from pkg/launcher/options.go, which will keep it from being a command-line/config flag. (We still need it here for all the control server setting stuff to work as expected -- this is the key it's stored under for the control server.)

Comment (directionless) [reply pkg/agent/flags/keys/keys.go:48]: I feel like we don't have great idioms yet.

**Review by directionless — APPROVED** [review]

**Pull request merged**